README.md

```diff
@@ -101,13 +98,14 @@ We are always happy to take whatever additional help we can get for this project
 - vista
 - Wolƒe Strider Shoσter
 - cabiste
-- zwexit (Orsell) 
 
 **Contributors:**
+- zwexit\ (Orsell) 
```

Comment:
bro really put his name above wanderer 😭

Reply:
My man, all i did was cut and paste it under contributor, it ain't that deep.